src/mpyl/steps/deploy/k8s/resources/sealed_secret.py

```diff
@@ -9,6 +9,7 @@
 
 class V1SealedSecret(CustomResourceDefinition):
     def __init__(self, name: str, secrets: dict[str, str]):
+        self.secrets = secrets
```

Comment:
Am I missing something or is this not doing anything, the value also doesn't seem to exist in the (super)class?

Reply:
hmm so the reason was that i dont necessarily want to deploy a sealedSecret that has no secrets inside. maybe there is a better way to check for this than to call `.secrets`  ?

Reply:
for example by checking `len(sealed_secret_refs)` ?

Reply:
did so in https://github.com/Vandebron/mpyl/pull/340/commits/3ffcb53463d845f5fc0f6bd056ab6e74b7db620c